src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import Button from './Button';
+
+const meta = {
+  title: 'Components/Button',
+  component: Button,
+  parameters: {
+    layout: 'centered'
+  },
+  argTypes: {
+    variant: {
+      options: ['', 'assertive', 'ghost', 'subtle'],
+      control: { type: 'radio' }
+    },
+    size: {
+      options: ['', 'small'],
+      control: { type: 'radio' }
+    },
+    icon: {
+      options: ['', 'plus', 'chevronRight', 'close'],
```

Comment:
Can we add a TODO here to look at what icons we want to support / how we want to handle that in storybook?